packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2130,10 +2130,7 @@ abstract class RenderObject extends AbstractNode implements HitTestTarget {
   // following (with the signature being whatever passes for coordinates
   // for this particular class):
   // bool hitTest(HitTestResult result, { Point position }) {
-  //   // If (x,y) is not inside this node, then return false. (You
-  //   // can assume that the given coordinate is inside your
-  //   // dimensions. You only need to check this if you're an
-  //   // irregular shape, e.g. if you have a hole.)
+  //   // If (x,y) is not inside this node, then return false.
```

Comment:
"You must not add yourself to /result/ if you return false"

^^ Is this part still accurate?